Makefile

```diff
@@ -104,9 +104,23 @@ secrets:
 		|| true; \
 	fi
 
+OPENTOK_VERSION = 2.10.0
+VERSION_FILE = Frameworks/OpenTok/version
+CURRENT_OPENTOK_VERSION = $(shell cat $(VERSION_FILE))
+ifeq ($(CURRENT_OPENTOK_VERSION),)
+	CURRENT_OPENTOK_VERSION = first
+endif
 opentok:
-	mkdir -p Frameworks/OpenTok
-	curl -s -N -L https://tokbox.com/downloads/opentok-ios-sdk-2.10.0 \
-		| tar -xz --strip 1 --directory Frameworks/OpenTok OpenTok-iOS-2.10.0/OpenTok.framework
+	if [ $(OPENTOK_VERSION) != $(CURRENT_OPENTOK_VERSION) ]; \
```

Comment:
if we put `@` at the beginning of this line then we can omit the statements from the logs when running `make opentok`. same with the other `if`